XenModel/Messages.resx

```diff
@@ -129,6 +129,9 @@
   <data name="ACTION_CHANGE_CONTROL_DOMAIN_MEMORY" xml:space="preserve">
     <value>Changing control domain memory settings on '{0}'</value>
   </data>
+  <data name="ACTION_CHANGE_CONTROL_DOMAIN_MEMORY_VALUE_TOO_LOW" xml:space="preserve">
+    <value>The memory {0} has to be at least the value of the static minimum ({1})</value>

```

Comment:
One more comment which did not occur to me earlier: `The memory 1GB has to be at least the value of the static minimum (2GB)` sounds a bit strange. How about  `The memory specified (1GB) was too low. It has to be at least the value of the static minimum (2GB)`